---
scorpius_manipulation/SceneObjectArray objects